super_team_picker_4/public/JS/shuffle.js

```diff
@@ -0,0 +1,61 @@
+function shuffle(list){
```

Comment:
not sure why the folder name is in capital. please be consistent with how you name other folders and files